CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
-## [_Unreleased_](https://github.com/freckle/freckle-app/compare/v1.10.3.0...main)
+## [_Unreleased_](https://github.com/freckle/freckle-app/compare/v1.11.0.0...main)
+
+## [v1.11.0.0](https://github.com/freckle/freckle-app/compare/v1.10.3.0...v1.11.0.0)
```

Comment:
I could see this not being a major bump, FWIW.

Reply:
I was on the fence and ended up with a major bump because of dropping using the `CIRCLECI` env var. I'll change it to a minor bump since using that env var is arguably an implementation detail, not a public API